### PR TITLE
[IMP] clouder: Add Docker travis support

### PR DESCRIPTION
**Pull request by lasley**
Let's see what happens!

### CODE REVIEW

**Comment by YannickB** [comment]
Looks like tests are failing :(

**Comment by lasley** [comment]
Yeah that saddened me. I got pissed off and greened all the independent repos first. Greening this one is a today/tomorrow task 😆

**Comment by lasley** [comment]
Alright with a little bit of luck, these next tests should go 🍏  & we will have ourselves working tests for the base

**Comment by lasley** [comment]
Woot! 

```
INFO: Start self test cycle
INFO: Testing /home/travis/build/LasLabs/clouder/clouder/images/base
DEBUG: Installing All Dependencies
DEBUG: Running Matrix Functions
Unable to find image 'lukasmartinelli/hadolint:latest' locally
latest: Pulling from lukasmartinelli/hadolint
51f5c6a04d83: Pulling fs layer
a3ed95caeb02: Pulling fs layer
2e9ce30f39de: Pulling fs layer
cd3ad359f777: Pulling fs layer
77372792ab13: Pulling fs layer
c3b0d09ac077: Pulling fs layer
6cce0d22f8d1: Pulling fs layer
cd3ad359f777: Waiting
77372792ab13: Waiting
c3b0d09ac077: Waiting
6cce0d22f8d1: Waiting
a3ed95caeb02: Verifying Checksum
a3ed95caeb02: Download complete
51f5c6a04d83: Verifying Checksum
51f5c6a04d83: Download complete
cd3ad359f777: Verifying Checksum
cd3ad359f777: Download complete
77372792ab13: Verifying Checksum
77372792ab13: Download complete
2e9ce30f39de: Verifying Checksum
2e9ce30f39de: Download complete
c3b0d09ac077: Verifying Checksum
c3b0d09ac077: Download complete
6cce0d22f8d1: Verifying Checksum
6cce0d22f8d1: Download complete
51f5c6a04d83: Pull complete
a3ed95caeb02: Pull complete
2e9ce30f39de: Pull complete
cd3ad359f777: Pull complete
77372792ab13: Pull complete
c3b0d09ac077: Pull complete
6cce0d22f8d1: Pull complete
Digest: sha256:3526a1231152cbe40c68d2372e40fd4b4f1e69d80e18ccb75e1bc050d30836bc
Status: Downloaded newer image for lukasmartinelli/hadolint:latest
INFO: Test passed for /home/travis/build/LasLabs/clouder/clouder/images/base
DEBUG: Running After Install
2017-01-08 18:22:24,788 7369 INFO openerp_test odoo.addons.clouder.tests.test_docker_clouder_base: test_tests (odoo.addons.clouder.tests.test_docker_clouder_base.TestDockerClouderBase)
INFO: Start self test cycle
INFO: Testing /home/travis/build/LasLabs/clouder/clouder/images/base
DEBUG: Installing All Dependencies
DEBUG: Sending build context to Docker daemon 8.704 kB
DEBUG: Step 1 : FROM alpine:3.5
DEBUG: 3.5: Pulling from library/alpine
DEBUG: 0a8490d0dfd3: Pulling fs layer
DEBUG: 0a8490d0dfd3: Verifying Checksum
DEBUG: 0a8490d0dfd3: Download complete
DEBUG: 0a8490d0dfd3: Pull complete
DEBUG: Digest: sha256:dfbd4a3a8ebca874ebd2474f044a0b33600d4523d03b0df76e5c5986cb02d7e8
DEBUG: Status: Downloaded newer image for alpine:3.5
DEBUG: ---> 88e169ea8f46
DEBUG: Step 2 : MAINTAINER Yannick Buron yannick.buron@gmail.com
DEBUG: ---> Running in 142f88a9f202
DEBUG: ---> ce1fbeff1d17
DEBUG: Removing intermediate container 142f88a9f202
DEBUG: Step 3 : RUN apk add --update ssmtp wget curl rsync bash
DEBUG: ---> Running in 4cfd5e942944
DEBUG: fetch http://dl-cdn.alpinelinux.org/alpine/v3.5/main/x86_64/APKINDEX.tar.gz
DEBUG: fetch http://dl-cdn.alpinelinux.org/alpine/v3.5/community/x86_64/APKINDEX.tar.gz
DEBUG: (1/15) Installing ncurses-terminfo-base (6.0-r7)
DEBUG: (2/15) Installing ncurses-terminfo (6.0-r7)
DEBUG: (3/15) Installing ncurses-libs (6.0-r7)
DEBUG: (4/15) Installing readline (6.3.008-r4)
DEBUG: (5/15) Installing bash (4.3.46-r4)
DEBUG: Executing bash-4.3.46-r4.post-install
DEBUG: (6/15) Installing ca-certificates (20161130-r0)
DEBUG: (7/15) Installing libssh2 (1.7.0-r2)
DEBUG: (8/15) Installing libcurl (7.52.1-r0)
DEBUG: (9/15) Installing curl (7.52.1-r0)
DEBUG: (10/15) Installing libattr (2.4.47-r4)
DEBUG: (11/15) Installing libacl (2.2.52-r2)
DEBUG: (12/15) Installing popt (1.16-r6)
DEBUG: (13/15) Installing rsync (3.1.2-r2)
DEBUG: (14/15) Installing ssmtp (2.64-r9)
DEBUG: (15/15) Installing wget (1.18-r1)
DEBUG: Executing busybox-1.25.1-r0.trigger
DEBUG: Executing ca-certificates-20161130-r0.trigger
DEBUG: OK: 15 MiB in 26 packages
DEBUG: ---> bee5ea1766c6
DEBUG: Removing intermediate container 4cfd5e942944
DEBUG: Step 4 : COPY ./docker-entrypoint.sh /
DEBUG: ---> 8ef7538db198
DEBUG: Removing intermediate container a9a7cfd3f86f
DEBUG: Step 5 : ENTRYPOINT /docker-entrypoint.sh
DEBUG: ---> Running in 3947e0dea4ca
DEBUG: ---> 7db04b505b44
DEBUG: Removing intermediate container 3947e0dea4ca
DEBUG: Step 6 : CMD cat
DEBUG: ---> Running in 6dc4ea1088e1
DEBUG: ---> cce3bd6abb91
DEBUG: Removing intermediate container 6dc4ea1088e1
DEBUG: Successfully built cce3bd6abb91
DEBUG: Running Matrix Functions
DEBUG: ['docker', 'run', '--name=latest-226', '-d', '-t', 'laslabs/docker-test:latest']
DEBUG: 76bb560b7689ad7ca18988c93afbbc469afbc6a2c385446cbef2492fc07c8b54
DEBUG: CONTAINER ID        IMAGE                        COMMAND                  CREATED             STATUS                  PORTS               NAMES
DEBUG: 76bb560b7689        laslabs/docker-test:latest   "/docker-entrypoint.s"   1 seconds ago       Up Less than a second                       latest-226
DEBUG: 0
DEBUG: Container ID 76bb560b7689ad7ca18988c93afbbc469afbc6a2c385446cbef2492fc07c8b54
DEBUG: Test dir /home/travis/build/LasLabs/clouder/clouder/images/base/tests
DEBUG: File test_entrypoint
INFO: Beginning test on /home/travis/build/LasLabs/clouder/clouder/images/base/tests/test_entrypoint
DEBUG: Command in 76bb560b7689ad7ca18988c93afbbc469afbc6a2c385446cbef2492fc07c8b54 responded with:
DEBUG: test
DEBUG: File test_command_running
INFO: Beginning test on /home/travis/build/LasLabs/clouder/clouder/images/base/tests/test_command_running
DEBUG: cat is running on 76bb560b7689ad7ca18988c93afbbc469afbc6a2c385446cbef2492fc07c8b54
INFO: Test passed for /home/travis/build/LasLabs/clouder/clouder/images/base
DEBUG: Running After Install
2017-01-08 18:22:29,215 7369 INFO openerp_test odoo.addons.clouder.tests.test_docker_clouder_base: Ran 2 tests in 52.514s
2017-01-08 18:22:29,215 7369 INFO openerp_test odoo.addons.clouder.tests.test_docker_clouder_base: OK
```

File: clouder/images/base/tests/test_command_running [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+# Copyright 2016 LasLabs Inc.
+# License MIT or later (https://opensource.org/licenses/MIT).
+
+# This script will test to see that $COMMAND is running on the Docker container.
```
Comment (lasley) [inline clouder/images/base/tests/test_command_running]: @YannickB - There's a few of these in the samples dir of DQT - https://github.com/LasLabs/docker-quality-tools/tree/master/sample_files/tests

Those are pretty much all I could think of while I was creating them, but I'm sure we'll need some others as we go along.

File: clouder/images/base/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM alpine:3.5
```
Comment (lasley) [inline clouder/images/base/Dockerfile]: Note latest fails lint. I stuck this at the current version, which is 3.5. 

This also outdates our tag, which is 3.4. We'll need to update that.

File: clouder/images/base/tests/test_command_running (hunk 1 above)
Comment (YannickB) [inline clouder/images/base/tests/test_command_running]: Yep looks good to me, you provided the basis, we will add the tests in the long run

**Comment by YannickB** [comment]
OK LGTM, I'm merging

**Pull request merged**